### ISSUE TITLE
Adding a "new client" results in undefined index in modal

### ISSUE THREAD

**Issue by franz-josef-kaiser** [justingreerbbi/wordpress-oauth-server#38]
In my local install, when doing a test ride with your plugin, I get the following error [here](https://github.com/justingreerbbi/wordpress-oauth-server/blob/6d8b733a74e70443a1d1d958a30098be4fc4141a/library/content/create-new-client.php#L131): 

> Notice: Undefined index: license in /var/www/dev/public/wp-content/plugins/oauth2-provider/library/content/create-new-client.php on line 131

The problem is that you assume in 

```php
if (!_vl($options['license']) && has_a_client())
```

that from `get_option('wo_options');` the `$options['license']` key is always set. Looking at the markup (no, I didn't debug this to the end), I can see the HTML form field:

```html
<input type="text" name="wo_options[license]" value="" length="40" style="width:300px;">
```

In short: If this is not filled, the other fields that depend on this should simply either have a `disabled` attribute or should not get displayed at all.

**Comment by justingreerbbi** [comment]
Fixed in prior versions. Please update. Closing this issue. If you feel this issue still exists, please open a new issue.
